client/db/src/bench.rs

```diff
@@ -544,7 +544,7 @@ impl<B: BlockT> StateBackend<HashFor<B>> for BenchmarkingState<B> {
 
 				if tracker.writes > 0 {
 					writes += 1;
-					repeat_writes += tracker.reads - 1;
+					repeat_writes += tracker.writes - 1;
```

Comment:
always better to extract stuff in their PR IMHO https://github.com/paritytech/substrate/pull/9653